integration/package.json

```diff
@@ -32,8 +34,8 @@
     "@angular/forms": "5.2.10",
     "@angular/platform-browser": "5.2.10",
     "@angular/platform-browser-dynamic": "5.2.10",
-    "@testeditor": "file:../dist/",
-    "@testeditor/messaging-service": "1.4.0",
+    "@testeditor": "file:../dist",
```

Comment:
legitimate use of 'file:…', would this go through the new pre-commit hook?

Reply:
pre-commit checks only package.json in the root of the project, all else goes unchecked